lib/tasks/projects.rake

```diff
@@ -40,14 +42,16 @@ namespace :projects do
 
     enqueued_projects_sum = 0
     project_ids_to_check.in_batches(of: batch_size) do |project_ids_batch|
+      enqueued_projects_sum += project_ids_batch.count
+
       project_ids_batch.pluck(:id).each_with_index do |project_id, i|
         CheckStatusWorker.perform_in(i, project_id)
       end
 
-      enqueued_projects_sum += project_ids_batch.count
       log_string = "#{enqueued_projects_sum} of up to #{max_num_of_projects_to_check} jobs were enqueued"
 
-      Rails.logger.info(log_string)
+      Rails.logger.info("logger: #{log_string}")
+      puts("puts: #{log_string}")
```

Comment:
Adding back the `puts` should be unnecessary but not harmful, and we weren't sure about whether the logging was working correctly, so this will help get more clarity.

Reply:
I've noticed this logger issue on this application before, I made a ticket to follow up. This is all good for now.